### PR TITLE
README: add quick start and installation instructions

### PR DESCRIPTION
**Pull request by orangecms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,29 @@ Using flattened Docker images from <https://github.com/u-root/sidecore-images>,
...
+
+### Installation
+
+To build the binary into your local Go bin directory, run:
+
+```sh
+go install github.com/u-root/sidecore/cmds/sidecore@latest
+```
+
+Ensure to have that bin directory in your `$PATH` to run it conveniently.
```
Comment (rminnich) [inline README.md]: do you want to tell them what that is, for people unused to go, i.e. say 
To build the binary into your local Go bin directory (usually ~/go/bin), run:

Comment (orangecms) [reply README.md]: I feel like all of the above is already a partial copy of upstream documentation... I'll add a link to that.

Comment (orangecms) [reply README.md]: It's also why I prefer the more common git clone way, which people are probably used to from other projects and is less implicit magic.

**Comment by orangecms** [comment]
rephrased again; how about that?

**Comment by orangecms** [comment]
well, now it is misleading.... because `GOBIN` is set to `''` by default, and the `$HOME/go/bin` is just one convention - the docs give more details

**Comment by orangecms** [comment]
from the docs:

> Executables (main packages) are installed to the directory named by the `GOBIN` environment variable, which defaults to `$GOPATH/bin` or `$HOME/go/bin` if the `GOPATH` environment variable is not set.

and I get:

```
$ go env | grep -E 'BIN|PATH'
GOBIN=''
GOPATH='/home/dama/go'
```

confusing at best

**Review by rminnich — APPROVED** [review]
nice.

**Pull request merged**